src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,199 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> addedNode = new Node<T>(value);
+        if (size > 0) {
+            addedNode.prev = tail;
+            tail.next = addedNode;
+            tail = addedNode;
+        }
+        ifEmptyList(addedNode);
+        size++;
```

Comment:
Just put removed logic here
```suggestion
        Node<T> node = new Node<T>(value);
        if (size > 0) {
            node.prev = tail;
            tail.next = node;
            tail = node;
        } else {
            head = node;
            tail = node;
        }
        size++;
```